src/halo/k8s/functions.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash -x
```

Comment:
What is diff between cluster-functions.sh and functions.sh. Why dont we merge them into one ? OR define them as say 
- aa-functions.sh 
- bb-functions.sh